addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -3,257 +3,234 @@
-            style="grid-area: 6 / 4 / 11 / 7; z-index: 4;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc4 g-height-5 g-col-lg-3 col-lg-3 justify-content-center text-center" data-name="Block"
+            style="grid-area: 6 / 4 / 11 / 7; z-index: 4; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-1) 100%);">
+            <h3>Community <br class="d-none d-lg-inline"/>Focus</h3>
+            <p>Building connections</p>
         </div>
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
+        <div class="o_grid_item o_grid_item_image g-height-10 g-col-lg-6 col-lg-6 text-center" data-name="Block"
             style="grid-area: 1 / 7 / 11 / 13; z-index: 5;">
             <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
         </div>
     </div>
 </template>
 
 <template id="s_masonry_block_images_template" groups="base.group_user">
-    <div class="row o_grid_mode" data-row-count="10">
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 1 / 11 / 7; z-index: 1;">
-            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
+    <div class="row o_grid_mode" data-row-count="8" style="gap: 16px;">
+        <div class="o_grid_item o_cc o_cc3 g-height-7 g-col-lg-6 col-lg-6 justify-content-start shadow rounded-3 overflow-hidden oe_img_bg o_bg_img_center" data-name="Block" style="z-index: 1; grid-area: 1 / 7 / 9 / 13; background-image: url(/web/image/website.s_masonry_block_default_image_1); --grid-item-padding-y: 30px; --grid-item-padding-x: 30px;">
+            <div class="o_we_bg_filter bg-black-50"/>
+            <h3 class="display-4">Innovation <br class="d-none d-lg-inline"/>Hub</h3>
+            <p class="lead">Where ideas come to life</p>
         </div>
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 7 / 11 / 13; z-index: 2;">
-            <img src="/web/image/website.s_masonry_block_default_image_2" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item o_cc o_cc3 g-height-7 g-col-lg-6 col-lg-6 justify-content-start shadow rounded-3 overflow-hidden oe_img_bg o_bg_img_center" data-name="Block" style="z-index: 2; grid-area: 1 / 1 / 9 / 7; background-image: url(/web/image/website.s_masonry_block_default_image_2); --grid-item-padding-y: 30px; --grid-item-padding-x: 30px;">
+            <div class="o_we_bg_filter bg-black-50"/>
+            <h3 class="display-4">Key <br class="d-none d-lg-inline"/>Milestone</h3>
+            <p class="lead">Reaching new heights together</p>
```

Comment:
> Remove the overflow-hidden class: 

It's needed if we want to use border-radius  +  `o_we_bg_filter` items  (overlays) 😕
![image](https://github.com/user-attachments/assets/97e26d19-c816-4ed5-92f1-493adcd9d64b)
![image](https://github.com/user-attachments/assets/973883c7-5328-4a7f-b4ba-ae6eb7bafb05)

For this specific template I'd keep the utility classes. It's unlikely that you'll ever have overflowing content here since these cards are super big. WDYT?

Reply:
Aah yes indeed, we have [a task](https://github.com/odoo/odoo/pull/163466) trying to fix this issue generally, but for the same reason, it was not approved yet :grimacing: 
Hmm, given that the only reason to not accept it is the unresponsiveness of the grid mode, I would say to remove the classes from the templates and to add back the CSS rule then :slightly_smiling_face: 
(I will definitely improve the grid responsiveness so this will never be an issue :+1: )